modules/local/ideel.nf

```diff
@@ -0,0 +1,24 @@
+process IDEEL {
+    label 'process_single'
+
+    container "quay.io/microbiome-informatics/ideel:1.0.0"
```

Comment:
If I'm not mistaked this container will fail, it doesn't have the tools that nextflow need to poke at the resourse usage.

I think the best it to move this script -> https://github.com/EBI-Metagenomics/containers/blob/master/ideel/ideel.py to this repo (we won't reuse elsewhere). Then use either bioconda or https://seqera.io/containers/ to create a container with matplotlib

Reply:
Well, the container has the bits it needs for Nextflow.

I still think we should remove that container, push this script into this pipeline bin folder and to create use a seqera container (or bioconda one if available) with matplotlib.